js/lib/jquery.bxslider/jquery.bxslider.css

```diff
@@ -0,0 +1,204 @@
+/**
```

Comment:
This will be added to the feature 'removing bxslider', which will be added to the next release.